x-pack/legacy/plugins/alerting/server/lib/alert_instance.ts

```diff
@@ -21,16 +21,16 @@ export class AlertInstance {
     this.meta = meta;
   }
 
-  shouldFire() {
-    return this.fireOptions !== undefined;
+  hasSecheduledActions() {
+    return this.scheduledExecutionOptions !== undefined;
   }
 
-  getFireOptions() {
-    return this.fireOptions;
+  getSechduledActionOptions() {
```

Comment:
s/`getSechduledActionOptions`/`getScheduledActionOptions`/

Reply:
Dammit, it's the laptop keyboard killing me

Reply:
It's a mouthful of a name!  Wonder if our names will start becoming more java-esque as we use TS and autocomplete saves our butts?